### PR TITLE
Finish basic mount support

### PR DESCRIPTION
**Pull request by goaaats**
Further steps would be checking flying, adding script hooks(or not when it's getting redesigned) and actually doing the quest that unlocks mounts correctly. There seem to be multiple modes for mounts(roaming/event), these will need to be handled correctly.

### CODE REVIEW

File: src/servers/Server_Zone/Network/Handlers/GMCommandHandlers.cpp [hunk 1]
```diff
@@ -298,6 +299,25 @@ void Core::Network::GameConnection::gm1Handler( const Packets::GamePacket& inPac
 
       break;
    }
+   case GmCommand::Mount:
+   {
+      if (param2 == 0)
```
Comment (itsMaru) [inline src/servers/Server_Zone/Network/Handlers/GMCommandHandlers.cpp]: style

File: src/servers/Server_Zone/DebugCommand/DebugCommandHandler.cpp [hunk 2]
```diff
@@ -397,6 +397,17 @@ void Core::DebugCommandHandler::add( char * data, Core::Entity::PlayerPtr pPlaye
       pPlayer->queuePacket(controlPacket);*/
 
    }
+   else if (subCommand == "unlock")
```
Comment (itsMaru) [inline src/servers/Server_Zone/DebugCommand/DebugCommandHandler.cpp]: style

File: src/servers/Server_Zone/DebugCommand/DebugCommandHandler.cpp [hunk 3]
```diff
@@ -397,6 +397,17 @@ void Core::DebugCommandHandler::add( char * data, Core::Entity::PlayerPtr pPlaye
...
+   {
+      int32_t id;
+      sscanf( params.c_str(), "%d", &id );
+
+      pPlayer->learnAction( id );
+   }
+   else if (subCommand == "enablecompanion")
```
Comment (itsMaru) [inline src/servers/Server_Zone/DebugCommand/DebugCommandHandler.cpp]: style

**Comment by SapphireMordred** [comment]
Still uses arbitraty numbers for initializing fields
```cpp
for( uint8_t i = 0; i < 255; i++ )
```

**Comment by SapphireMordred** [comment]
Closing for eventual reimplementation

**Pull request closed without merging**